src/functionalTest/java/uk/gov/hmcts/reform/userprofileapi/CreateUserProfileFuncTest.java

```diff
@@ -30,7 +32,6 @@
 
     @Value("${targetInstance}") private String targetInstance;
 
-    @Autowired private Environment environment;
     @Autowired private FuncTestRequestHandler testRequestHandler;
```

Comment:
Does sonar allow annotations on the same line? sure it pull me up on this, or maybe it allows in test code, worth a check

Reply:
I think it allows it in tests as I think some of them are in ia-case-api.
https://github.com/hmcts/ia-case-api/blob/master/src/functionalTest/java/uk/gov/hmcts/reform/iacaseapi/EndpointSecurityTest.java#L34